docs/sources/administration/configuration.md

```diff
@@ -435,6 +435,10 @@ The length of time that Grafana maintains idle connections before closing them.
 
 If enabled and user is not anonymous, data proxy will add X-Grafana-User header with username into the request. Default is `false`.
 
+### response_limit
+
+Adds limit for the response bytes, after http data source request. Default is `1000000`.
```

Comment:
```suggestion
Adds limit for the response bytes after an HTTP data source request. Default is `1000000`.
```

Reply:
@achatterjee-grafana, that's an outdated version, pushed some commits after that, could you please check again? 🙏 
Thanks!